### PR TITLE
ignore [v6r16] DynamicMonitoringDB

### PR DESCRIPTION
**Pull request by zmathe**
Added a module for handling the insertion, query and deletion of dynamic component monitoring data into/from the elasticsearch database.

Note that this module requires the ElasticSearchDB and ElasticDB modules being worked on in Zoltan's fork.

### CODE REVIEW

File: FrameworkSystem/DB/DynamicMonitoringDB.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+from DIRAC.Core.Base.ElasticDB import ElasticDB
+from DIRAC import gLogger, gConfig, S_OK, S_ERROR
+
+class DynamicMonitoringDB( ElasticDB ):
+  """
+  This class manages logs stored in Elasticsearch
+  """
+
+  def __init__( self, name = 'Framework/ComponentMonitoringDB' ):
+    ElasticDB.__init__( self, 'ComponentMonitoringDB', name )
```
Comment (chaen) [inline FrameworkSystem/DB/DynamicMonitoringDB.py]: use of super ?

File: FrameworkSystem/DB/DynamicMonitoringDB.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+      query[ 'bool' ][ 'must' ].append( { 'match': { key : matchFieldsDict[ key ] } } )
+
+    return query
+
+  def insertLog( self, log ):
+    """
+    Creates a new log entry in ElasticSearch
+    :param dict log: Should be a dictionary containing the logging information to be stored
+    """
+    if not 'timestamp' in log:
```
Comment (chaen) [inline FrameworkSystem/DB/DynamicMonitoringDB.py]: @zmathe  at least we now know that there is a timestamp field in the logs :D

Comment (zmathe) [inline FrameworkSystem/DB/DynamicMonitoringDB.py]: yes, because we store time series. :)

**Comment by zmathe** [comment]
recommendations applied

**Comment by zmathe** [comment]
This can be closed later...

**Comment by fstagni** [comment]
Why? Where is it inserted?

**Comment by zmathe** [comment]
Because I am redoing the DynamicMonitoring,  I keep the relevant PRs which I have not marked (can be closed...)

**Comment by zmathe** [comment]
MonitoringSystem will provide this functionality

**Pull request closed without merging**